md/src/arithmetic/practice/SumAverageRunningInt.java

```diff
@@ -0,0 +1,19 @@
+package io.codelex.arithmetic.practice;
+
+public class SumAverageRunningInt {
+    public static void main(String[] args) {
+        int sum = 0;
+        double average;
+        int lowerBound = 1;
+        int upperBound = 100;
+
+        for (int number = lowerBound; number <= upperBound; number++) {
+            sum += number;
+        }
+        average = sum/(double)upperBound;
```

Comment:
Šis gan īsti nestrādās, ja mēs lower bound nomainītu, piemēram uz 30 - tad upperBound nebūtu vienāds ar skaitļu skaitu un arī vidējais nebūtu pareizs.